src/mpvis/mpdfg/dfg_builder.py

```diff
@@ -13,8 +13,11 @@ def __init__(self, dfg, log, parameters):
         self.parameters = parameters
 
     def start(self):
-        sorted_log = self.log.sort_values(by=[self.parameters.timestamp_key])
-        sorted_log = self.log.sort_values(by=[self.parameters.start_timestamp_key])
+        sorting_order = []
```

Comment:
[REQ]
No need to construct list dynamically with appends. Just do:

```suggestion
        sorting_order = [self.parameters.start_timestamp_key, self.parameters.timestamp_key]
```